src/packages/v2v3/components/V2V3Project/ProjectDashboard/components/SuccessPayView/components/SuccessPayCard.tsx

```diff
@@ -8,7 +8,7 @@ import { useProjectPageQueries } from 'packages/v2v3/components/V2V3Project/Proj
 import { useMemo } from 'react'
 import { twMerge } from 'tailwind-merge'
 import { formatCurrencyAmount } from 'utils/format/formatCurrencyAmount'
-import { ProjectHeaderLogo } from '../../ProjectHeader/components/ProjectHeaderLogo'
+import { ProjectHeaderLogo } from '../../../../../../../../components/Project/ProjectHeader/ProjectHeaderLogo'
```

Comment:
can probably clean these up

```suggestion
import { ProjectHeaderLogo } from 'components/Project/ProjectHeader/ProjectHeaderLogo'
```

Reply:
Oh yep missed some, got it.